driver/src/main/scala/core/actors.scala

```diff
@@ -1245,29 +1245,13 @@ trait MongoDBSystem extends Actor {
 }
 
 @deprecated("Internal class: will be made private", "0.11.14")
-final class LegacyDBSystem private[reactivemongo] (
```

Comment:
I was trying to avoid having to define 3 DbSystems but no problem if you'd prefer that. Any ideas for a name X509DbSystem?